src/ray/gcs/gcs_server/gcs_actor_manager.h

```diff
@@ -125,6 +125,10 @@ class GcsActor {
   /// Get the mutable ActorTableData of this actor.
   rpc::ActorTableData *GetMutableActorTableData();
 
+  UniqueID GetActorWorkerAssignmentID();
```

Comment:
could this be `UniqueID GetActorWorkerAssignmentID() const;`?

Reply:
Yes, will fix.